corehq/apps/motech/openmrs/urls.py

```diff
@@ -0,0 +1,23 @@
+from django.conf.urls import url
+from corehq.apps.motech.openmrs import views
+
+urlpatterns = [
+    # concepts
+    url(r'^rest/concept/$',
+        views.all_openmrs_concepts,
+        name='all_openmrs_concepts'),
+    url(r'^rest/concept/search/$',
+        views.concept_search,
+        name='openmrs_concept_search'),
+    url(r'^rest/concept/sync/$',
+        views.sync_concepts,
+        name='openmrs_sync_concepts'),
+
+    # patients
```

Comment:
I think you're right and I can just get rid of the whole `patients` portion. It's copied from an earlier prototype that did use it, but I can just add it back once it becomes a thing.